common/repositories.sh

```diff
@@ -177,59 +164,6 @@ transact_repositories=(
   https://github.com/hyperledger/transact-sdk-go.git
 )
 
-iroha_repositories=(
-  https://github.com/hyperledger/iroha-android.git
-  https://github.com/hyperledger/iroha-deploy.git
-  https://github.com/hyperledger/iroha-dotnet.git
-  https://github.com/hyperledger/iroha-ed25519.git
-  https://github.com/hyperledger/iroha-ios.git
-  https://github.com/hyperledger/iroha-java.git
-  https://github.com/hyperledger/iroha-javascript.git
-  https://github.com/hyperledger/iroha-python.git
-  https://github.com/hyperledger/iroha-scala.git
-  https://github.com/hyperledger/iroha.git
-)
-
-indy_repositories=(
-  https://github.com/hyperledger/indy-agent.git
-  https://github.com/hyperledger/indy-crypto.git
```

Comment:
were the iroha, indy, composer, cello, and quilt stanzas deleted on purpose? I must be missing something

Reply:
They were duplicated.

Reply:
Wait...why am I not seeing them anymore

Reply:
Diff doesn’t show them, but when I look at the file, they are there.